MBBSEmu/HostProcess/Structs/UidxrefStruct.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace MBBSEmu.HostProcess.Structs
+{
+    /// <summary>
+    ///     User-id cross reference structure
+    ///
+    ///     MAJORBBS.H
+    /// </summary>
+    public class UidxrefStruct
+    {
+        public const int XRFSIZ = 15; // user-id search string size
+        public const int UIDSIZ = 30; // user-id size (including trailing zero)
+        public const int SPARESIZ = 6; // spare
+        
+
+        public byte[] xrfstg
+        {
+            get
+            {
+                ReadOnlySpan<byte> xrfstgSpan = Data;
+                return xrfstgSpan.Slice(0, XRFSIZ + 1).ToArray();
```

Comment:
You can one line these getters as `get => new ReadOnlySpan<byte>(Data).Slice(xyz).ToArray();`